src/Runner/DefaultTestResultCache.php

```diff
@@ -24,7 +24,7 @@ final class DefaultTestResultCache implements \Serializable, TestResultCache
     /**
      * Provide extra protection against incomplete or corrupt caches
      *
-     * @var array<string, string>
+     * @var int[]
```

Comment:
Note: the original annotation saying that this array contains strings was wrong.

Reply:
Nice catch! The early prototypes did use `string` descriptions